app/src/ui/diff/diff-options.tsx

```diff
@@ -116,9 +117,7 @@ export class DiffOptions extends React.Component<
         decoration={PopoverDecoration.Balloon}
         onClickOutside={this.closePopover}
       >
-        <h3 id="diff-options-popover-header">
-          Diff {__DARWIN__ ? 'Settings' : 'Options'}
-        </h3>
+        <h3 id="diff-options-popover-header">{header}</h3>
```

Comment:
This isn't necessary... NVDA correctly announces it as "Diff Options" but VoiceOver was doing "Diff level 2 settings". I assume because the `{__DARWIN__ ? 'Settings' : 'Options'}` was adding in a detectable child element.. This just make the VoiceOver announcement a clean  `Diff Settings`